### PR TITLE
Symfony 5.4 + PHP 8.1

### PR DESCRIPTION
**Pull request by ruudk**
Best to review commit by commit.

Symfony 6 will be added in another PR, as it has a blocker:
- https://github.com/malukenho/docheader/pull/35

### CODE REVIEW

**Comment by ruudk** [comment]
@othillo Tests are not running. Can you setup branch protection so that it's visible that it requires approval?

**Comment by othillo** [comment]
> @othillo Tests are not running. Can you setup branch protection so that it's visible that it requires approval?
Yeah we only run them for PRs to master. Feel free to cherry-pick my first commit and rebase to master.

**Comment by ruudk** [comment]
~Why only on master? Every PR should be tested, right?~

Wow, didn't even realize it targetted it like this. Didn't choose to do that! Magic ✨

**Comment by ruudk** [comment]
@othillo Ready for review.

Symfony 6 will need a separate PR.

File: test/Command/CommandMetadataEnricherTest.php [hunk 1]
```diff
@@ -14,26 +14,52 @@
...
+use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class CommandMetadataEnricherTest extends TestCase
 {
+    /**
+     * @var MyCommand
+     */
     private $command;
```
Comment (othillo) [inline test/Command/CommandMetadataEnricherTest.php:30]: we might as well use typed properties here now that we are on PHP 7.4

**Review by othillo — APPROVED** [review]

File: test/Command/CommandMetadataEnricherTest.php (hunk 1 above)
Comment (ruudk) [reply test/Command/CommandMetadataEnricherTest.php:30]: I was temped to do so, but I don't want to change things that are not needed as the PR is already huge.

**Comment by othillo** [comment]
Thank you very much, @ruudk !

File: test/Command/CommandMetadataEnricherTest.php (hunk 1 above)
Comment (othillo) [reply test/Command/CommandMetadataEnricherTest.php:30]: Good call!

**Pull request merged**